app/src/main/java/com/wednesday/bankingdetails/utility/Utils.kt

```diff
@@ -0,0 +1,11 @@
+package com.wednesday.bankingdetails.utility
+
+import java.util.*
+
+class Utils {
+    companion object {
+        fun capitalizeFirstLetter(string: String) =
```

Comment:
Can you move this to an extension function on String

Reply:
Already done in next PR